src/RadixDlt.NetworkGateway.GatewayApi/gateway-api-schema.yaml

```diff
@@ -3004,6 +3072,14 @@ components:
         - Pending
         - Rejected
 
+    TransactionSubintentStatus:
```

Comment:
MINOR: Again, to avoid confusion with TransactionIntentStatus, can we call this just SubintentStatus?